engine.py

```diff
@@ -507,9 +507,13 @@ def save_to_path(self, document, path):
         
             (_, ext) = os.path.splitext(path)
             ext = ext.lower()
-
-            # these extensions are supported by photoshop      
-            if ext ==".bmp":
+          
+            if ext == ".psb":
+                self.adobe._save_as_psb(self.adobe.File(path))
```

Comment:
We shouldn't call a protected method. We can make it public if need be. I also think the argument is incorrect. So we should do this instead:

```python
self.adobe.save_as_psb(path)
```